bindings/python/examples/wallet/logger.py

```diff
@@ -1,8 +1,12 @@
-from iota_sdk import Wallet, StrongholdSecretManager, init_logger, CoinType, ClientOptions
-from dotenv import load_dotenv
 import json
 import os
 
+from dotenv import load_dotenv
+
+# pylint: disable=no-name-in-module
```

Comment:
?

Reply:
This is the same case as here:
https://github.com/iotaledger/iota-sdk/pull/1003/files#r1295416035